model/project_parser.go

```diff
@@ -465,12 +468,15 @@ func LoadProjectForVersion(v *Version, identifier string, shouldSave bool) (*Pro
 
 	// TODO: don't need separate ppFromDB variable once UseParserProject = true
 	if shouldSave && ppFromDB == nil {
-		if err = pp.Insert(); err != nil {
+		// we upsert here instead of insert bc it is possible that since this function started a project has been inserted, and we don't want to error in that case
+		// https://mongodb.splunkcloud.com/en-US/app/search/search?q=search%20index%3Devergreen%20%22error%20inserting%20parser%20project%22&display.page.search.mode=smart&dispatch.sample_ratio=1&earliest=1575954000&latest=1576645200&sid=1576609915.8744045
```

Comment:
We shouldn't include private links in a public repo. (You might have been planning to remove since this is a WIP, but wanted to call it out anyway just in case.)

Reply:
Gotcha, yeah this was just a temporary reference for "this is the proof of the thing I want to fix".